### ISSUE TITLE
Add tags for a visually-impaired member

### ISSUE THREAD

**Issue by brashley46** [diaspora/diaspora#7391]
A friend of mine who just joined cannot see the add tags box in the manage tags list at her main page. She is visually challenged, and teh screen magnifier she uses takes the column left off screen. She tried mobile mode and it does not have that option. What to do?

**Comment by goobertron** [comment]
Diaspora should look for a proper solution for users in such cases. However, an interim solution for your friend, could be to use Firefox and an add-on such as [Zoom Page](https://addons.mozilla.org/en-GB/firefox/addon/zoom-page/) or [NoSquint](https://addons.mozilla.org/en-GB/firefox/addon/nosquint-plus/), which will enable her to zoom just the text, not the entire page. This means that at high magnification the page will render as if for small screens, in a single-column view).

(I wanted to include a screenshot, but unfortunately GitHub won't let me upload an image at this moment.)

NoSquint additionally allows (or at least, used to) screen and text colours to be changed if this helps visibility for a user. I stopped using this add-on when the developer changed and switched to Zoom Page, so I can't recommend the current version of NoSquint, but it might still be good.

I hope that helps her in the meantime while a proper solution is being worked on.

**Comment by svbergerem** [comment]
In the mobile version you can search for tags (enter `#hashtag`) and once you are on the tags page there is a button to follow / unfollow a tag. In the followed tags submenu in the mobile drawer there is also a “Manage followed tags” page where you can see a list of your followed tags and you can unfollow them there, too.

**Comment by svbergerem** [comment]
In the desktop version I tried the GNOME screen magnifier and that worked fine. Which screen magnifier is your friend using? Does it remove the whole left sidebar or is there specific problem with adding a hashtag?

By the way in the desktop version you can also search for tags, go to the tags page and follow / unfollow the hashtag there.

**Comment by brashley46** [comment]
Thanks, I have forwarded your comments to her ... I'm not even sure which operating system she is using, as she lives rather far away from here and we have never physically met.

**Comment by brashley46** [comment]
But neither of your suggestions, svbergerem, allow her to get to the add tags box and generate her own tags. The Firefox add-ons that goobertron mentioned might work better. Thanks all!

**Comment by svbergerem** [comment]
@brashley46 

> But neither of your suggestions, svbergerem, allow her to get to the add tags box and generate her own tags.

You can just search for #hashtagthatdoesntexistyet and when you click the “Follow” button the server will create the tag if it doesn't exist yet. The text input in the left sidebar doesn't add any functionality. It just makes it easier to add tags when you are already on the followed tags stream page.

**Comment by DeusFigendi** [comment]
> Diaspora should look for a proper solution for users in such cases. (…) which will enable her to zoom just the text, not the entire page. (…) additionally allows (or at least, used to) screen and text colours to be changed if this helps visibility for a user.

Could this be done by a color theme for visual disabled people (except for blinds of course) or do they really only affect the colors and not the whole css?

**Comment by denschub** [comment]
> Could this be done by a color theme for visual disabled people

No. [This](https://developer.mozilla.org/en-US/docs/Web/Accessibility) is a good starting place for stuff we should probably do eventually.

**Comment by SkewsMe** [comment]
[Is Your Website Accessible? If Not, You Could Be Violating the ADA](https://premium.wpmudev.org/blog/website-accessibility-ada/)

[Web Content Accessibility Guidelines (WCAG) 2.0](https://www.w3.org/TR/WCAG20/)
